uavcan/equipment/power/ECBInputChannel.uavcan

```diff
@@ -0,0 +1,18 @@
+#
+# Nested type.
+# ECB's input channel status
+#
+
+#
+# Current status of channel
+#
+float16 voltage # Output voltage on channel
+float16 current # Current through channel
+
+uint2 ERROR_FLAG_OVERCURRENT    = 1 # Channel current is above maximum
+uint2 error_flags
```

Comment:
Please make the next field byte-aligned. Perhaps throw in more bits to `error_flags`?